taskcluster/ci/startup-test/kind.yml

```diff
@@ -0,0 +1,61 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+job-defaults:
+    description: Run a simple UI test to verify Nightly build can start 
+    treeherder:
+        kind: test
+        platform: 'nightly-start-test/opt'
+        tier: 2
+    worker-type: b-android
+    worker:
+        docker-image: {in-tree: ui-tests}
+        max-run-time: 7200
+        env:
+            GOOGLE_APPLICATION_CREDENTIALS: '.firebase_token.json'
+            GOOGLE_PROJECT: moz-fenix
+        artifacts:
+            - name: public
+              path: /builds/worker/artifacts
+              type: directory
+    run:
+        using: run-commands
+        use-caches: false
+        secrets:
+            - name: project/mobile/fenix/firebase
+              key: firebaseToken
+              path: .firebase_token.json
+              json: true
+    run-on-tasks-for: []
+    attributes:
+        build-type: debug
+        code-review: true
```

Comment:
Nit: Remove this line. It's not useful anymore now that we handle nightly builds.